src/openboardview/TCL.h

```diff
@@ -29,9 +29,13 @@
 
 #include "BoardView.h"
 #include <regex>
+#include <atomic>
 
 //#include "TCLDUMMY.h"
 
+#ifdef OBV_USE_POPPLER
+
+#error Enabling pdf rendering with poppler changes the license of openboardview. Uncomment this line to continue with this derivation of openboardview under GPL license
```

Comment:
Shouldn't this say "Comment-out this line" rather than "Uncomment"?

Reply:
I see you later committed a change commenting out this line. If this is how we're notifying users this error prompt should be un-commented when merged.

That said, ideally it'd be better if there was a prompt created by the CMAKE buildscript - something like "type yes to confirm you understand". Ideally the build script would also bundle a copy of the GPL into the build.

Reply:
i just leave poppler enabled while i am working on the code. since a merge into openboardview is not in the immediate future, i don't care about it at the moment.

don't care about the cmake option. anyone able to build openboardview from source should have no problem following simple instructions. even more true about anyone planning to do a closed source derivative work.

Reply:
yes. comment it out. for sure. sorry bout that.